applications/crossbar/priv/couchdb/schemas/callflows.route_to_cid.json

```diff
@@ -0,0 +1,44 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "_id": "callflows.route_to_cid",
+    "description": "Endpoints lookup by cid number",
+    "properties": {
+        "can_call_self": {
+            "description": "Toggle whether devices of the same owner can call each other",
+            "type": "boolean"
+        },
+        "can_text_self": {
+            "description": "Toggle whether devices of the same owner can text each other",
+            "type": "boolean"
+        },
+        "cid_types": {
```

Comment:
let's add the items schema:

```json
"cid_types":{
    "type":"array",
    "items":{
        "type":"string",
    }
}
```

Reply:
Another thought - set `minItems` to `1` to make sure no empty arrays (and remove `default:[]`)